### PR TITLE
No-item: template

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1581
[Problem] No-item template is not implemented
[Solution] Template implemented according to guideline

[Screenshot]
image

    
Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: examples/mobile/UIComponents/components/no-item/no-item-main-tab.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
```
Comment (HunseopJeong) [inline examples/mobile/UIComponents/components/no-item/no-item-main-tab.html:1]: Which file uses this?

File: examples/mobile/UIComponents/components/no-item/index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+					<option value="dark" class="ui-dropdown-two-lines">
+						Dark Theme
+					</option>
+				</select>
+			</div>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview ui-content-area">
+				<li class="ui-li-anchor">
+					<a href="no-item-main-tab.html">
```
Comment (TomaszLukawskiSam) [inline examples/mobile/UIComponents/components/no-item/index.html:36]: The `no-item-main-tab.html` is using here.

Comment (HunseopJeong) [reply examples/mobile/UIComponents/components/no-item/index.html:36]: Okay,, I didn't see it.

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Comment by TomaszLukawskiSam** [comment]
The PR has been rebased. Merge conflicts need resolved.

**Pull request merged**